tutorials/viewer.md

```diff
@@ -1,58 +1,296 @@
+
+Here is a demo of interactivity for the `Image` and `Points` layers in the example above.
+
+![image](resources/layer_control_panel.gif)
+
+### new layer buttons
+New `Points`, `Shapes`, and `Labels` layers can be added to the viewer using the new layer buttons in the bottom righthand corner of the gui. These correspond to the calls
+
+```python
+viewer.add_points(data)
+viewer.add_shapes(data)
+viewer.add_labels(data)
+```
+but with empty data. Once added in the gui these layers become accessible in the layers list and at `viewer.layers`.
+
+Layers can also be deleted by selecting them and the clicking on the trash icon, the far right button, or by dragging the layers and dropping them into the trash.
+
+In the console a layer at index `i` can be removed by
+```python
+viewer.layers.pop(i)
+```
+
+Adding and deleting layers from the gui looks as follows
+
+![image](resources/layer_new.gif)
+
+Notice here how we added two `Points` layers and used the layer properties widget to change their colors. We then added a `Shapes` layer which allowed us to draw some rectangles, before we deleted all our new layers.
+
+### status bar
+
+At the very bottom of the gui there is a status bar that contains useful updates and tips.
+
+On the lefthand side of the status bar there is a message that contains information about the position of the mouse and the values of any images or the indices of any points that are currently hovered over, depending on which layer is selected. The status bar will also display information about what button you are clicking in the layer control panel too.
+
+The righthand side of the status bar contains some helpful tips depending on which layer and tools are currently selected.
+
+You should be able to see examples of the status bar changing as the cursor moves in our examples above.
+
+## dimension sliders
+One of the main strengths of **napari** is that it has been designed from the beginning to handle n-Dimensional data. While much consumer photography is 2D and `RGB`, scientific image data can often be volumetric (i.e. 3D), volumetric timeseries (i.e. 4D), or even higher dimensional. **napari** places no limits on the dimensionality of its input data for all of its layer types.
+
+Adding data with a dimensionality greater than 2D will cause dimension sliders to appear directly underneath the main canvas and above the status bar. As many sliders as needed will appear to ensure the data can be fully browsed. For example a 3D dataset needs one slider, a 4D dataset needs two sliders, and so on.
+
+It is also possible to mix data of different shapes and dimensionality in different layers. If a 2D and 4D dataset are both added to the viewer then the sliders will only affect the 4D dataset and the 2D dataset will be remain the same.
```

Comment:
"Effectively, the two datasets are broadcast together using [NumPy broadcasting rules](https://docs.scipy.org/doc/numpy/user/basics.broadcasting.html)"

Reply:
added to the end of the sentence